server/trafficrules/src/main/java/com/coffeejug/trafficrules/projection/ProgressPrj.java

```diff
@@ -0,0 +1,6 @@
+package com.coffeejug.trafficrules.projection;
+
+public interface ProgressPrj {

```

Comment:
The use of abbreviations makes code less readable. I would suggest to avoid it

Reply:
ok